CONTRIBUTING.md

```diff
@@ -78,6 +78,21 @@ The following GitHub users are the currently available sponsors, either by being
 * [@djaglowski](https://github.com/djaglowski)
 
 Whenever a sponsor is picked from the top of this list, please move them to the bottom.
+
+## Adding New Metrics
+Following these steps for contributing additional metrics.
```

Comment:
It might be worth adding a note that this workflow only applies to receivers that use mdatagen

Reply:
added a note